functions/starlark/set_namespace.star

```diff
@@ -0,0 +1,10 @@
+# set the namespace on each resource
```

Comment:
Is this documented in the catalog? What prompted this? Also we've discussed adding a toolchain column to the catalog  (Can be follow up after the move).

Reply:
This is not documented in the catalog, I will add it once it's committed. This was prompted by [this issue](https://github.com/GoogleContainerTools/kpt/issues/683) a user filed noting that there was no easy way to add the namespace field to unstructured configs in order to use them with Config Sync.

The new PR will add a toolchain column to the catalog [here](https://github.com/prachirp/kpt/blob/update-general-docs/site/content/en/guides/consumer/function/catalog/_index.md)